parkings/tests/api/enforcement/test_permit.py

```diff
@@ -104,3 +104,26 @@ def test_permit_data_matches_permit_object(staff_api_client, permit):
     check_permit_object_keys(response.data)
     check_permit_subject_keys(response.data['subjects'][0])
     check_permit_areas_keys(response.data['areas'][0])
+
+
+def test_permit_bulk_create_creates_cache_items(staff_api_client, permit_series):
+    permit_data = [
+        {
+            "series": permit_series.id,
+            "external_id": generate_external_ids(),
+            "subjects": generate_subjects(),
+            "areas": generate_areas(),
+        },
+        {
+            "series": permit_series.id,
+            "external_id": generate_external_ids(),
```

Comment:
This could conflict with the previously created external ID. Use static values instead to avoid such conflicts, e.g. `"E1"` for the first external_id and `"E2"` for the second.